user-interface/src/main/java/life/qbic/datamanager/AppConfig.java

```diff
@@ -135,30 +136,43 @@ public UserInformationService userInformationService(UserRepository userReposito
   }
 
   @Bean
-  public ProjectAccessGrantedPolicy projectAccessGrantedPolicy(CommunicationService communicationService,
+  public ProjectAccessGrantedPolicy projectAccessGrantedPolicy(
+      life.qbic.projectmanagement.application.communication.EmailService emailService,
       JobScheduler jobScheduler, UserInformationService userInformationService,
       AppContextProvider appContextProvider) {
-    var informUserAboutGrantedAccess = new InformUserAboutGrantedAccess(communicationService, jobScheduler,
+    var informUserAboutGrantedAccess = new InformUserAboutGrantedAccess(emailService,
+        jobScheduler,
         userInformationService, appContextProvider);
     return new ProjectAccessGrantedPolicy(informUserAboutGrantedAccess);
   }
 
   @Bean
   public BatchRegisteredPolicy batchRegisteredPolicy(
-      CommunicationService communicationService, ProjectAccessService accessService,
+      life.qbic.projectmanagement.application.communication.EmailService emailService, ProjectAccessService accessService,
       UserInformationService userInformationService, AppContextProvider appContextProvider,
       JobScheduler jobScheduler) {
-    var informUsers = new InformUsersAboutBatchRegistration(communicationService, accessService,
+    var informUsers = new InformUsersAboutBatchRegistration(emailService, accessService,
         userInformationService, appContextProvider, jobScheduler);
     return new BatchRegisteredPolicy(informUsers);
   }
 
   @Bean
-  public CommunicationService communicationService(@Value("${spring.mail.host}") String host,
+  public EmailProvider emailProvider(@Value("${spring.mail.host}") String host,
       @Value("${spring.mail.port}") int port, @Value("${spring.mail.username}") String mailUserName,
       @Value("${spring.mail.password}") String mailUserPassword) {
-    MailServerConfiguration mailServerConfiguration = new MailServerConfiguration(host, port,
+    var mailServerConfiguration = new life.qbic.infrastructure.email.MailServerConfiguration(
+        host, port,
         mailUserName, mailUserPassword);
-    return new EmailCommunicationService(mailServerConfiguration);
+    return new EmailProvider(mailServerConfiguration);
+  }
+
+  @Bean
+  public EmailService identityEmailService(EmailProvider emailProvider) {
+    return new IdentityEmailProvider(emailProvider);
+  }
+
+  @Bean
+  public life.qbic.projectmanagement.application.communication.EmailService projectEmailService(EmailProvider emailProvider){
+    return new ProjectManagementEmailProvider(emailProvider);
```

Comment:
Why different interfaces here?

Reply:
because they come from different application cores (with different domains).  Did you check the documentation I provided? :)